core/src/test/java/org/apache/calcite/test/RelOptRulesTest.java

```diff
@@ -1350,7 +1350,6 @@ private void basePushFilterPastAggWithGroupingSets(boolean unchanged)
             + "(select * from emp e1 union all select * from emp e2) r2");
   }
 
-  @Ignore("cycles")
```

Comment:
I've no idea what is the expected output for the tests, however those tests do not cycle anymore, so I activate them